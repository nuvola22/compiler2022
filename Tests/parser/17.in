var a: array[0..10] of Integer;
  
begin writeln(a[0]); end.