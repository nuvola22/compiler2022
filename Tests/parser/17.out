      program
         variable
            a
            array
               range
                  0
                  10
               Integer
         begin
            call
               writeln
               array access
                  a
                  0
